stable/atlantis/templates/statefulset.yaml

```diff
@@ -157,6 +208,22 @@ spec:
                 key: bitbucket_secret
           {{- end }}
           {{- end }}
+          {{- if .Values.ARM_CLIENT_SECRET }}
```

Comment:
lets just add a new value `env` that takes in an array of arbitrary environment variables

Reply:
working on this..

Reply:
Looks like that's already in place... https://github.com/helm/charts/blob/a0a3ef4bdccdf8f582185afea7c2095bce16fdcc/stable/atlantis/templates/statefulset.yaml#L79-L82

Reply:
That was removed in a commit this morning